response_operations_ui/views/collection_exercise.py

```diff
@@ -1162,17 +1163,22 @@ def view_sample_ci_summary(short_name: str, period: str) -> str:
 @collection_exercise_bp.route("/<short_name>/<period>/view-sample-ci/summary/<form_type>", methods=["GET"])
 @login_required
 def view_ci_versions(short_name: str, period: str, form_type: str) -> str:
-    survey = survey_controllers.get_survey_by_shortname(short_name)
-    survey_ref = survey.get("surveyRef")
+    redis_cache = RedisCache()
+    survey = redis_cache.get_survey_by_shortname(short_name)
```

Comment:
This gives us minor gains at this stage, as the subsequent `build_collection_exercise_details` then calls the same (non cache) method again anyway. A separate card will be progressed to improve that flow.